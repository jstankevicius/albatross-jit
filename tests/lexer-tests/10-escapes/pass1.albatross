#strings can have special escape chars
"this \n string \n has \n multiple \n lines"
