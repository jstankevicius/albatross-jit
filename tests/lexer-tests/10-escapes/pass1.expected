1 2 STRING 38 this 
 string 
 has 
 multiple 
 lines
